src/deluge/gui/views/view.cpp

```diff
@@ -953,8 +955,11 @@ void View::modEncoderButtonAction(uint8_t whichModEncoder, bool on) {
 }
 
 void View::setKnobIndicatorLevels() {
-	if (!getRootUI()) {
-		return; // What's this?
+	if (!getRootUI()
+	    || (getCurrentUI() == &automationInstrumentClipView && //don't update knob indicator levels
```

Comment:
nit: make `automationInstrumentClipView` check a separate check below, no need to re-use the mysterious `return`